common/src/test/scala/org/neo4j/spark/util/Neo4jImplicitsTest.scala

```diff
@@ -209,4 +209,51 @@ class Neo4jImplicitsTest {
     val actual = input.flattenKeys()
     Assert.assertEquals(expected, actual)
   }
+
+  @Test
+  def `should create a nested Java map`(): Unit = {
```

Comment:
```suggestion
  def `should deserialized dotted/stringified map into a nested Java map`(): Unit = {
```
ok, maybe the name is too long :P

Reply:
it's fine, done